dockerfiles/dataset/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
+MAINTAINER Madhuri Yechuri <madhuri.yechuri@clusterhq.com>
```

Comment:
Should be `ClusterHQ <contact@clusterhq.com>`  to be in line with our package metadata
See https://github.com/ClusterHQ/flocker/blob/master/admin/packaging.py#L784